libbzip2-rs-sys-cdylib/src/lib.rs

```diff
@@ -1,3 +1,17 @@
+#![no_std]
+
 extern crate libbzip2_rs_sys;
 
+use core::ffi::c_int;
+use core::panic::PanicInfo;
 pub use libbzip2_rs_sys::*;
+
+#[panic_handler]
+fn panic_handler(_info: &PanicInfo) -> ! {
+    extern "C" {
+        fn bz_internal_error(errcode: c_int);
+    }
+
+    unsafe { bz_internal_error(-1) }
```

Comment:
can you change the `bz_internal_error` to exit? right now it would loop forever here

```
extern void bz_internal_error(int errcode) {
    fprintf(stderr, "bzip2 hit internal error code: %d\n", errcode);
}
```